### ISSUE TITLE
Query error in AllegroGraph v7.1.0 rc1

### ISSUE THREAD

**Issue by mdorf** [ncbo/goo#110]
Error:

```
Executing query failed:
keyword list (:where (:modifiers :where (:bgp #(!<http://data.bioontology.org/submission_statuses/UPLOADED> {type} !<http://data.bioontology.org/metadata/SubmissionStatus>))) :default-base "http://127.0.0.1:37515/repositories/ontoportal" :query-string "ASK FROM <http://data.bioontology.org/metadata/SubmissionStatus> WHERE { <http://data.bioontology.org/submission_statuses/UPLOADED> a <http://data.bioontology.org/metadata/SubmissionStatus> . }" :from ((:from . !<http://data.bioontology.org/metadata/Submissio…
```

Query:

```
ASK FROM <http://data.bioontology.org/metadata/SubmissionStatus> 
WHERE { 
  <http://data.bioontology.org/submission_statuses/UPLOADED> a <http://data.bioontology.org/metadata/SubmissionStatus> . 
}
```

Contacted AllegroGraph support, opened a ticket: spr45252

**Comment by mdorf** [comment]
From AllegroGraph support:

> Thanks for reporting this as well! This seems to be a regression in AG
> 7.0.0 release, so it is a bit weird that you only see it on
> 7.1.0. I'll test 7.0.* versions tomorrow to see if I'm missing
> something. The fix will be available in AG 7.0.4 as well.

**Comment by mdorf** [comment]
We've deployed the official AG 7.0.4 build to the OntoPortal Virtual Appliance. Currently testing to make sure this issue has been addressed.

**Comment by mdorf** [comment]
This issue has been fixed in the latest build of AllegroGraph  v7.0.4.
